st2actions/st2actions/runners/mistral/v2.py

```diff
@@ -147,7 +147,7 @@ def run(self, action_parameters):
 
         if not is_workbook:
             # Non-workbook definition containing multiple workflows is not supported.
-            if len([k for k, v in six.iteritems(def_dict) if k != 'version']) != 1:
+            if len([k for k, _ in six.iteritems(def_dict) if k != 'version']) != 1:
```

Comment:
local linter complained so I fixed it. It would seem that my linter is more strict in some regards than `make flake8`. 

To be clear this change is unrelated.